adobe_vendor_id.py

```diff
@@ -231,3 +277,141 @@ def account_id_and_label(self, delegated_patron_identifier):
     def urn_to_label(self, urn):
         """We have no information about patrons, so labels are sparse."""
         return "Delegated account ID %s" % urn
+
+
+class VendorIDAuthenticationError(Exception):
+    """The Vendor ID service is working properly but returned an error."""
+
+    
+class VendorIDServerException(Exception):
+    """The Vendor ID service is not working properly."""
+    
+
+class AdobeVendorIDClient(object):
```

Comment:
This code is moved in from `AdobeVendorIDAcceptanceTestScript`, which now uses it. I don't have unit tests for it, but I have re-tested it against the library registry, and I have written unit tests for the way it's used in the controller above.